Libplanet.Tests/TestUtils.cs

```diff
@@ -234,6 +236,12 @@ public static byte[] GetRandomBytes(int size)
             );
         }
 
+        public static HashDigest<T> MakeRandomHashDigest<T>()
+            where T : HashAlgorithm
+        {
+            return new HashDigest<T>(GetRandomBytes(HashDigest<T>.Size));
+        }
+
```

Comment:
I believe this *should not* exist because `HashDigest<T>` implies its values are *deterministic* and cryptographically derived… *not arbitrary*.  If its purpose you intended is just for unit tests I rather suggest you to make an extension method in the *Libplanet.Tests* project.  `TestUtils` could be a good place.